torchvision/datasets/fvgc_aircraft.py

```diff
@@ -0,0 +1,119 @@
+import os
+import shutil
+from typing import Any, Callable, List, Optional, Tuple
+
+import PIL.Image
+
+from .utils import download_and_extract_archive, verify_str_arg
+from .vision import VisionDataset
+
+
+class FVGCAircraft(VisionDataset):
```

Comment:
Minor typo :)
I think we'll also need to update other parts like the test class, the docstring below, etc.


```suggestion
class FGVCAircraft(VisionDataset):
```